crates/types/src/motion_command.rs

```diff
@@ -82,6 +86,13 @@ pub enum HeadMotion {
     Unstiff,
 }
 
+#[derive(Clone, Copy, Debug, Default, Deserialize, PartialEq, Serialize, SerializeHierarchy)]
+pub enum PixelTarget {
```

Comment:
Maybe `ImageRegionTarget`? Pixel target feels like it should correspond to the exact pixels

Reply:
Good point! Done.